packages/block-library/src/latest-posts/edit.js

```diff
@@ -309,6 +309,9 @@ class LatestPostsEdit extends Component {
 							'trim',
 						] );
 						let excerpt = post.excerpt.rendered;
+						if ( post.excerpt.raw !== '' ) {
```

Comment:
The raw except will probably contain unfiltered information, why are we using the raw version?

Reply:
Is the non-raw version trimmed? If yes I think that is a rest API bug.

Reply:
it may be but that's what I found raw to be currently, the manually added excerpt.

Reply:
And post.excerpt.rendered does not contain the manual excerpt?

Reply:
Not when I tested for that, I'll re-check just to make double sure :)

Reply:
I can have a plugin that applies some changes to the excerpt using the filter the_excerpt. For example, my plugin changes all the excerpts to be equal to "Excerpt". In the front end latest posts will render the excerpts as "Excerpt" but on the preview, if we rely on the raw content we will not render "Excerpt" and we will render something else making the preview wrong.
I think we can not rely on raw for the editor preview because the filters are not applied so we don't know if the excerpts are what raw contains.
Why not just keep some logic but rely on the rendered version? e.g: remove this if?

Reply:
Hmm, yes you are correct. The problem I hit when I implemented is that post.excerpt.rendered is badly served by the API to be the trimmed post version even if there is a manual excerpt.

Maybe I missed something, will check again.

Reply:
Ok, re tested and it appears I was wrong. The rendered version respects manual excepts! So I removed the conditional. Thanks for insisting :)